services/simcore/docker-compose.yml

```diff
@@ -642,6 +642,20 @@ services:
         - traefik.http.middlewares.${SWARM_STACK_NAME_NO_HYPHEN}_payments_auth.basicauth.users=${TRAEFIK_USER}:${TRAEFIK_PASSWORD}
         - traefik.http.routers.${SWARM_STACK_NAME}_payments.middlewares=${SWARM_STACK_NAME_NO_HYPHEN}_payments_auth, ${SWARM_STACK_NAME_NO_HYPHEN}_sslheader
 
+  dynamic-scheduler:
```

Comment:
it's stateless I presume, no persistant volumes?

Reply:
correct